src/annotation.js

```diff
@@ -598,6 +605,8 @@ var pointAnnotation = function (args) {
 };
 inherit(pointAnnotation, annotation);
 
+registerAnnotation('point', pointAnnotation, {point: true});
```

Comment:
My only suggestion would is that features (keys) are not visible anywhere else. Should we create a list of it somewhere at the top may be? then you can say annotation.point: true